modules/ROOT/pages/_partials/sync/data-sync-case-study-generic.adoc

```diff
@@ -0,0 +1,29 @@
+[id="data-sync-case-study-generic-{context}"]
```

Comment:
spoke to @wei-lee  need to remove this case study

Reply:
Complete in this commit: https://github.com/aerogear/mobile-docs/pull/691/commits/48a7e8cda0cbedc0b2f7e5bc5e91730af1cb6899